engines/bastion_katello/app/assets/javascripts/bastion_katello/hosts/host.factory.js

```diff
@@ -21,6 +21,15 @@ angular.module('Bastion.hosts').factory('Host',
         resource.prototype.hasSubscription = function () {
             return angular.isDefined(this.subscription_facet_attributes) && angular.isDefined(this.subscription_facet_attributes.uuid);
         };
+
+        resource.prototype.isRpmEnabled = function() {
+            return !this.isDebEnabled();
+        };
+
+        resource.prototype.isDebEnabled = function() {
+            return angular.isDefined(this.operatingsystem_name) && this.operatingsystem_name.includes("Debian");
```

Comment:
U..... :sneezing_face:

Reply:
Yeah... any suggestions? it doesn't appear there is a better way. We could modify the Foreman API to send the operatingsystem 'family' and make some more direct comparisons here rather than matching strings.

Reply:
Well, if we modified the foreman-api, it could as well deliver the desired package format.

Reply:
Oh, i think, you got me wrong... i meant this also needs to work with Ubuntu.

Reply:
I get it now! Gesundheit :laughing:

Reply:
Updated!